src/main/java/com/evuv/expressions/CollectionValueExpression.java

```diff
@@ -0,0 +1,43 @@
+package com.evuv.expressions;
+
+import com.evuv.exceptions.EventBindingException;
+
+import java.util.Collection;
+import java.util.Map;
+
+public class CollectionValueExpression <T extends Collection> implements Expression<T>, BindedExpression<T> {
```

Comment:
Can we define also the Generic of the collection ?
<T extends Collection \<U\>, U>